packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -141,6 +142,7 @@ describe('RpcContractProvider test', () => {
       transferTicket: jest.fn(),
       increasePaidStorage: jest.fn(),
       updateConsensusKey: jest.fn(),
+      smartRollupOriginate: jest.fn(),
```

Comment:
Will update with a smartRollupOriginate test suite when available, right?

Reply:
thanks for the catch thought i added a test already